packages/veritone-json-schemas/schemas/vtn-standard/object/object.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://docs.veritone.com/schemas/vtn-standard/object.json",
+  "title": "vtn-standard.object",
+  "description": "Standard engine output for Object at Veritone",
+  "type": "object",
+  "properties": {
+    "validationContracts": {
+      "type": "array",
+      "items": {
+        "const": "object"
+      }
+    },
+    "object": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "object": {
```

Comment:
Doesn't this mean something like this?
```
{
  "object": [{
    "object": {
      // objectResult definition goes here
    }
  }]
}
```

And if so, what's wrong with the tests that allows this to work?

Reply:
Can I get a response on this?  I'm particularly concerned on this one